package.json

```diff
@@ -28,6 +28,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@emotion/react": "11.7.1",
+    "@mui/material": "5.3.1",
+    "@typescript-eslint/eslint-plugin": "5.10.1",
```

Comment:
I think they are there because of the eslint rule, do you want me to turn off that rule?
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md